CLAUDE.md

```
@@ -1 +1 @@
-After making any changes always execute `pnpm nb && pnpm test` to verify
+- After making any changes always execute `pnpm build && pnpm test:coverage` to verify
```

markdownlint-cli2 (0.18.1)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)